src/main/scala/viper/carbon/modules/impls/DefaultHeapModule.scala

```diff
@@ -774,10 +774,6 @@ class DefaultHeapModule(val verifier: Verifier)
   def resetBoogieState: Stmt = {
     Havoc(heapVar)
   }
-  def initOldState: Stmt = {
```

Comment:
I'm confused. Was this never called in the first place? If it was , shouldn't there a call somewhere that can be removed?

Reply:
It was never called in the first place. Also, it is not defined in any of the traits that `DefaultHeapModule` implements. The code for initalizing the old state is contained in `DefaultStateModule.initOldState`, which does not invoke `initOldState` on the components.